tools/export/e2studio/__init__.py

```diff
@@ -14,35 +14,34 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from os.path import splitext, basename
+from tools.export.gnuarmeclipse import GNUARMEclipse
 
-from tools.export.exporters import Exporter, deprecated_exporter
-
-@deprecated_exporter
-class E2Studio(Exporter):
+class E2Studio(GNUARMEclipse):
     NAME = 'e2 studio'
     TOOLCHAIN = 'GCC_ARM'
 
     TARGETS = [
         'RZ_A1H',
     ]
 
+    # override
     def generate(self):
-        libraries = []
-        for lib in self.resources.libraries:
-            l, _ = splitext(basename(lib))
-            libraries.append(l[3:])
-
-        ctx = {
-            'name': self.project_name,
-            'include_paths': self.resources.inc_dirs,
-            'linker_script': self.resources.linker_script,
+
+        jinja_ctx = self.collect_tmpl_vars()
             
-            'object_files': self.resources.objects,
-            'libraries': libraries,
-            'symbols': self.toolchain.get_symbols()
-        }
-        self.gen_file('e2studio/%s_project.tmpl' % self.target.lower(), ctx, '.project')
-        self.gen_file('e2studio/%s_cproject.tmpl' % self.target.lower(), ctx, '.cproject')
-        self.gen_file('e2studio/%s_gdbinit.tmpl' % self.target.lower(), ctx, '.gdbinit')
-        self.gen_file('e2studio/launch.tmpl', ctx, '%s OpenOCD.launch' % self.project_name)
+        print
+        print 'Create a e2 studio C++ managed project'
+        print 'Project name: {0}'.format(self.project_name)
+        print 'Target: {0}'.format(self.toolchain.target.name)
+        print 'Toolchain: {0}'.format(self.TOOLCHAIN)
+
+        self.gen_file('e2studio/%s_project.tmpl' % self.target.lower(), jinja_ctx, '.project', trim_blocks=True, lstrip_blocks=True)
+        self.gen_file('e2studio/%s_cproject.tmpl' % self.target.lower(), jinja_ctx, '.cproject', trim_blocks=True, lstrip_blocks=True)
+        self.gen_file('e2studio/%s_gdbinit.tmpl' % self.target.lower(), jinja_ctx, '.gdbinit')
+        self.gen_file('e2studio/launch.tmpl', jinja_ctx, '%s OpenOCD.launch' % self.project_name, trim_blocks=True, lstrip_blocks=True)
+
+        self.gen_file('gnuarmeclipse/mbedignore.tmpl', jinja_ctx, '.mbedignore')
+        self.gen_file('gnuarmeclipse/makefile.targets.tmpl', jinja_ctx, 'makefile.targets', trim_blocks=True, lstrip_blocks=True)
+
+        print
+        print 'Done. Import the \'{0}\' project in Eclipse.'.format(self.project_name)
```

Comment:
Could you change this to say `"Import the project located at \'{0}\'.format(self.export_dir)`? That way the user knows exactly what to do.